imgui-SFML.cpp

```diff
@@ -547,11 +555,7 @@ void Image(const sf::Texture& texture,
 void Image(const sf::Texture& texture, const sf::Vector2f& size,
     const sf::Color& tintColor, const sf::Color& borderColor)
 {
-    GLuint nativeHandle = texture.getNativeHandle();
-    ImTextureID textureID;
-    std::memcpy(&textureID, &nativeHandle, sizeof(GLuint));
-
-    ImGui::Image(textureID, size, ImVec2(0, 0), ImVec2(1, 1), tintColor, borderColor);
+    ImGui::Image(convertGLTextureHandleToImTextureID(texture.getNativeHandle()), size, ImVec2(0, 0), ImVec2(1, 1), tintColor, borderColor);
```

Comment:
Can you make separate variables for textureID here and in other calls? Right now it's a bit hard to read
After that, I'll merge :)

Reply:
No problem!